src/main/kotlin/io/github/mojira/arisa/modules/RemoveVersionModule.kt

```diff
@@ -14,6 +14,7 @@ class RemoveVersionModule(
 ) : Module {
     override fun invoke(issue: Issue, lastRun: Instant): Either<ModuleError, ModuleResponse> = with(issue) {
         Either.fx {
+            assertGreaterThan(affectedVersions.size, 1).bind()

```

Comment:
Maybe it would be saner to verify that when the amount of removed versions is `< affectedVersions`, otherwise the same issue might occur again when two versions are added at the same time.

So assert that `removeAddedVersions.size < affectedVersions.size`.

Reply:
Good point, fixed

Reply:
Really minor, but might be better to compare with `removeAddedVersions`. Otherwise the module would refuse to remove versions when the versions are not part of the affected versions anymore (and would therefore not be part of `removeAddedVersions` either).

Reply:
Yep, and I think that was causing the error. Thanks